airflow/configuration.py

```diff
@@ -110,7 +110,11 @@ def run_command(command):
         'non_pooled_task_slot_count': 128,
     },
     'operators': {
-        'default_owner': 'airflow'
+        'default_owner': 'airflow',
+        'default_cpus': 1,
```

Comment:
I think cpus (and gpus) is OK for shorthand (FWIW I stole the naming spec from the open-source mesos scheduler aurora).